tuned/plugins/plugin_irqbalance.py

```diff
@@ -0,0 +1,108 @@
+from . import base
+from .decorators import command_custom
+import tuned.logs
+import errno
+import perf
+
+log = tuned.logs.get()
+
+class IrqbalancePlugin(base.Plugin):
+	"""
+	Plugin for irqbalance settings management.
+	"""
+
+	def __init__(self, *args, **kwargs):
+		super(IrqbalancePlugin, self).__init__(*args, **kwargs)
+		self._cpus = perf.cpu_map()
+		self._irqbalance_sysconfig = "/etc/sysconfig/irqbalance"
```

Comment:
Make into a constant in a similar way such as `consts.GRUB2_CFG_FILES` for example?

Reply:
It's a good suggestion, thanks. However, given that this is a plugin-specific thing, I think it's cleaner to keep it in the plugin. But I should probably make it into a class variable. @yarda What do you think?

Reply:
It could be class variable, but I would initialize it from the consts. It's not optimal, but we currently do it this way, e.g.:
```
...
# modules plugin configuration
MODULES_FILE = "/etc/modprobe.d/tuned.conf"

# systemd plugin configuration
SYSTEMD_SYSTEM_CONF_FILE = "/etc/systemd/system.conf"
...
```
```

Reply:
Thanks. I've changed it to use consts now.